tests/specs/controllers/OpenEAppController.test.js

```diff
@@ -114,15 +117,15 @@ describe('OpenEAppController', () => {
         sandbox.restore();
     });
 
-    it('should prevent viewing the page if user is not logged in', async () => {
+    it('should redirect to sign in page if user is not logged in', async () => {
         // when
         sandbox.stub(HelperService, 'getUserData').callsFake(() => ({
             loggedIn: false,
         }));
         await OpenEAppController.renderPage(reqStub, resStub);
 
         // then
-        expect(resStub.serverError.called).to.be.true;
+        expect(resStub.redirect.firstCall.args[0]).to.equal('http://localhost/3000//sign-in?eappid=test_unique_app_id');
```

Comment:
I notice a `//` at the start of the path. I know it the path gets clean by might be worth fixing.

Reply:
Good spot, I'll sort that out.